dotcom-rendering/src/components/SignInGate/signInGate.ts

```diff
@@ -15,16 +16,19 @@ import type { SignInGateTestMap } from './types';
 export const signInGateTests: ReadonlyArray<ABTest> = [
 	signInGateMainVariant,
 	signInGateMainControl,
+	signInGateTimesOfDay,
 ];
 
 export const signInGateTestVariantToGateMapping: SignInGateTestMap = {
 	'main-control-5': gateMainControl,
 	'main-variant-5': gateMainVariant,
+	morning: gateMainVariant,
 };
 
 // Component Id does not need to match gate test name, as ab test info passed separately to ophan
 // Consider Id name relevant to the gate component or design. Use snake_case
 export const signInGateTestIdToComponentId: { [key: string]: string } = {
 	SignInGateMainVariant: 'main_variant_5',
 	SignInGateMainControl: 'main_control_5',
+	SignInGateTimesOfDay: 'main_variant_5',
```

Comment:
Is this variant name accurate?

Reply:
yeah, it's because it is using the same design as the main component. The comment above explains what this map is doing. I know it's looks odd to introduce this line out of context.